services/grpc/grpc.js

```diff
@@ -250,6 +254,12 @@ class ZapGrpc extends EventEmitter {
       // Set up the subscription.
       const { serviceName, methodName } = this.availableSubscriptions[key]
       const service = this.services[serviceName]
+
+      if (!service) {
+        grpcLog.warn(`gRPC subscription "${key}" not available.`)
```

Comment:
Maybe worth logging some additional data here too just incase:
`Attempt to initialize ${serviceName} has failed. gRPC subscription "${key}"  not available.`
Since the service is accessed via `this.services[serviceName]` this would give us more debug info.